### DISCUSSION TITLE
Interactive Search function like Prowlarr

### DISCUSSION THREAD

**Discussion by Ttfgggf** [autobrr/autobrr#1827] [Ideas]
Allow ad-hoc search requests made either via RSS or IRC.

**Comment by KyleSanderson** [comment]
Have you checked the Releases page? are you looking for something else?

**Reply by Ttfgggf** [reply]
> Have you checked the Releases page? are you looking for something else?

Yes. I have checked the release page but it dosen't work like Prowlarr Interactive Search.

I believe it makes use of HTTP to make queries to whatever trackers are configured via the use of Torznab.

Where you can put anything in the search box and it will search all the monitored trackers.
